pkgs/applications/misc/squeekboard/default.nix

```diff
@@ -0,0 +1,84 @@
+{ stdenv
+, pkgs
+, fetchFromGitLab
+, cmake
+, meson
+, ninja
+, pkgconfig
+, gnome3
+, glib
+, gtk3
+, wayland
+, wayland-protocols
+, rustc
+, cargo
+, libcroco
+, libxml2
+, libxkbcommon
+, rustPlatform
+, makeWrapper
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "squeekboard";
+  version = "1.9.1";
+
+  src = fetchFromGitLab {
+    domain = "source.puri.sm";
+    owner = "Librem5";
+    repo = pname;
+    rev = "334898c5a5d45a5883654b4f2f07f28ac8f2bfc4";
+    sha256 = "1nrihpffr91p3ipif5k5z9vcvkwxk6qy8hp93w13s7gnwl2g93hx";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkgconfig
+    rustc
+    cargo
+    glib.dev
+    gtk3.dev
+    wayland
+    gnome3.gnome-desktop
+    makeWrapper
+  ];
+
+  buildInputs = [
+    glib.dev
+    gtk3.dev
+    gnome3.gnome-desktop
+    wayland
+    wayland-protocols
+    libcroco
+    libxml2
+    libxkbcommon
+  ];
+
+  cargoSha256 = "1fkhj4i2l2hdk9wvld6ryvnm1mxfwx3s555r7n42pg9f5namn1sr";
+
+  NIX_CFLAGS_COMPILE = "-I${glib.dev}/include/gio-unix-2.0";
```

Comment:
Please don't use `NIX_CFLAGS_COMPILE` - this should be passed to meson.

Reply:
This is likely an upstream bug and should be fixed there: https://github.com/NixOS/nixpkgs/issues/36468

Reply:
Done https://github.com/NixOS/nixpkgs/issues/36468#issuecomment-637131546